docs/embeddable-uis/element/access-requests.mdx

```diff
@@ -0,0 +1,589 @@
+---
+sidebar_position: 2
+title: Access Requests API Documentation
+---
+import HelpSupportTile from "@site/src/components/elements/HelpSupportTile.js";
+import ActionContainer from "@site/src/components/elements/ActionContainer.js";
+import ActionTile from "@site/src/components/elements/ActionTile.js";
+import CodeDropdown from "@site/src/components/elements/CodeDropdown.js";
+import FlexWrapper from "@site/src/components/elements/FlexWrapper.js";
+import code from "@site/src/components/elements/codeBlock.json";
+
+# Access Requests API Documentation
+
+This documentation provides details about the Access Requests API.
+
+You can also find information about the Access Requests API in the [Permit Redoc](https://api.permit.io/v2/redoc).
+## Overview
+
+The Access Requests API allows you to manage access requests in your application .
+Access requests enable your users to request access to specific roles.
```

Comment:
Elaborate more - using access request api you can create access requests for specific roles and even to a specific resource instances within your application, moreover, it provides the relevant moderators - based on your decision, to approve or deny requests from users.

Reply:
And then explain that you are showing now an example of how to use the API.

Reply:
fixed